### PR TITLE
Logged in navigation

### PR DESCRIPTION
**Pull request by dzucconi**
Closes https://github.com/aredotna/ervell/issues/793

dropdown-expandable

### CODE REVIEW

**Comment by dzucconi** [comment]
Ah woops, Bound the click to the wrong thing :)

File: components/logged_in_navigation/components/you/index.jade [hunk 1]
```diff
@@ -7,7 +7,7 @@
     )
 
 .Dropdown.js-dropdown
-  .DropdownSection
+  .DropdownSection.DropdownSection--expandable.js-expandable( data-key='you', data-state='active' )
     a.DropdownItem.DropdownItem--primary( href="/#{user.get('slug')}" )
```
Comment (broskoski) [inline components/logged_in_navigation/components/you/index.jade]: This link is getting the `preventDefault` treatment, no? Does it make sense to add a link to the profile directly? cc @danpia

Comment (broskoski) [inline components/logged_in_navigation/components/you/index.jade]: Talked with @danpia in person, just add a link underneath this to the user profile, with the label `Profile`.

Comment (dzucconi) [inline components/logged_in_navigation/components/you/index.jade]: Done.

**Pull request merged**